source/about/cloud-shared.rst

```diff
@@ -0,0 +1,85 @@
+
+Mattermost supports a **multi-AZ (availability zones)** strategy in the same site/region.
+
+Daily backups of the database, object storage, and high availability clusters are captured and retained for 30 days.
+
+In addition, highly available observability tools with automated alerting, long term metrics, and logs retention are retained for a duration of 1 year.
+
+Security
+~~~~~~~~
+
+You have access to all the resources required to run the Mattermost application with the highest security standards, including data encryption at rest and in transit.
+
+Your pre-configured cluster is secure by default, based on industry best practices including Data encryption at rest and in transit, TLS certificates life cycle management, and automatic security updates.
+
+Mattermost maintains control over network and security policies, including `encryption <#encryption>`__, database, data, object storage, backup schedules, and compliance certifications. 
+
+Authentication and authorization
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Mattermost offers advanced security and authentication options for integrating with corporate directories, including :doc:`Active Directory/LDAP </onboard/ad-ldap>`, Okta, OneLogin, :doc:`SAML </onboard/sso-saml>`, :doc:`Google </onboard/sso-google>`, :doc:`EntraID </onboard/sso-office>`, and :doc:`OpenID </onboard/sso-openidconnect>`.
+
+Secure networking
+~~~~~~~~~~~~~~~~~~
+
+Enterprise customers with a Mattermost Cloud Shared deployment can :ref:`configure IP filtering <manage/cloud-ip-filtering:configure ip filtering>` through CIDR-based IP ranges, within the Mattermost System Console to specify authorized IPs or IP ranges for seamless access control. Users attempting to access the workspace from IPs outside defined ranges are restricted from entry.
+
+Encryption
+~~~~~~~~~~~
+
+Mattermost provides encryption-in-transit and encryption-at-rest capabilities. Mattermost supports :doc:`TLS encryption </install/setup-tls>`, including AES-256 with 2048-bit RSA on all data transmissions, between Mattermost client applications and the Mattermost server. You may either set up TLS on the Mattermost Server or :doc:`install a proxy such as NGINX </install/proxy-to-mattermost-transport-encryption>`, and set up TLS on the proxy.
+
+Connections to :doc:`Active Directory/LDAP </onboard/ad-ldap>` can :ref:`optionally be secured with TLS or stunnel <configure/authentication-configuration-settings:ad/ldap port>`.
+
+Connections to calls are secured with a combination of:
+
+- TLS: The existing WebSocket channel is used to secure the signaling path.
+- DTLS v1.2 (mandatory): Used for initial key exchange. Supports ``TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256`` and ``TLS_ECDHE_ECDSA_WITH_AES_256_CBC_SHA`` algorithms.
+- SRTP (mandatory): Used to encrypt all media packets (i.e. those containing voice or screen share). Supports ``AEAD_AES_128_GCM`` and ``AES128_CM_HMAC_SHA1_80`` algorithms.
+
+Gossip encryption (experimental)
```

Comment:
@stylianosrigas we are not supporting this today right? A good idea would be how we can make the gossip encryption to use AWS KMS.

Reply:
We don't support it you are right ;)

Reply:
Removed the Gossip section content.